File: packages/sdk-java/src/main/java/com/zenvio/sdk/model/EmailCancelResponse.java
package com.zenvio.sdk.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class EmailCancelResponse {
    private boolean success;
    private Data data;

    public boolean isSuccess() { return success; }
    public void setSuccess(boolean success) { this.success = success; }
    public Data getData() { return data; }
    public void setData(Data data) { this.data = data; }

    @JsonIgnoreProperties(ignoreUnknown = true)
    public static class Data {
        private String emailId;
        private String status;

        @com.fasterxml.jackson.annotation.JsonProperty("email_id")
        public String getEmailId() { return emailId; }
        @com.fasterxml.jackson.annotation.JsonProperty("email_id")
        public void setEmailId(String emailId) { this.emailId = emailId; }
        public String getStatus() { return status; }
        public void setStatus(String status) { this.status = status; }
    }
}

File: packages/sdk-dotnet/src/Zenvio/Models/Email/EmailCancelResponse.cs
using System.Text.Json.Serialization;

namespace Zenvio.Models.Email;

public record EmailCancelResponse(
    [property: JsonPropertyName("success")] bool Success,
    [property: JsonPropertyName("data")] EmailCancelData Data
);

public record EmailCancelData(
    [property: JsonPropertyName("email_id")] string EmailId,
    [property: JsonPropertyName("status")] string Status
);
